stdlib/toolchain/CompilerPluginSupport/CMakeLists.txt

```diff
@@ -0,0 +1,41 @@
+#===--- CMakeLists.txt - Compiler plugin support library -------------------===#
+#
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2021 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See https://swift.org/LICENSE.txt for license information
+# See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+#
+#===------------------------------------------------------------------------===#
+
+set(library_name "swift_CompilerPluginSupport")
+
+add_swift_target_library("${library_name}" SHARED IS_STDLIB
```

Comment:
This is a host library, not a target library. I recommend following what we did with the `ASTGen` library, which we've hit with a hammer enough times to get it building Swift code that links into the compiler.

Reply:
Fixed